modules/application_permissions/main.tf

```diff
@@ -137,7 +137,7 @@ resource "aws_iam_user_policy_attachment" "this" {
 
 module "workspace_user_policy" {
   source  = "guidion-digital/helper-workspace-policy/aws"
-  version = "~> 2.0"
```

Comment:
Should be ~> 2.0, read: https://developer.hashicorp.com/terraform/language/expressions/version-constraints#-3.

Now you're not allowing a version after 2.4. But 2.4 has not been released yet in here: https://github.com/guidion-digital/terraform-aws-helper-workspace-policy/actions/runs/11576272838/job/32224966606?pr=22